### ISSUE TITLE
Unnecessary duplicate implementation for loading assets

### ISSUE THREAD

**Issue by loopier** [loopier/animatron#33]
I think we have implemented loading assets twice, once in `CommandInterface.gd` (https://github.com/loopier/animatron/blob/ef4574bead90b153bea7951ac2324dd986aa392b/CommandInterface.gd#L616) and another one in `assets_helper.gd` (https://github.com/loopier/animatron/blob/ef4574bead90b153bea7951ac2324dd986aa392b/asset_helpers.gd#L149).

I believe we're only calling the one in `CommandInterface.gd`.

**Comment by totalgee** [comment]
Yeah, I don't know why the (presumably newer?) one in `CommandInterface` exists. I assume the `asset_helpers` one is brought over from A-G3? (We do call its counterpart, `assetHelpers.loadSprites()`, for sprites, at least.)
